app/helpers/admin/claims_helper.rb

```diff
@@ -5,9 +5,9 @@ module ClaimsHelper
     def admin_eligibility_answers(eligibility)
       [].tap do |a|
         a << [t("student_loans.questions.admin.qts_award_year"), academic_years(eligibility.qts_award_year)]
-        a << [t("student_loans.questions.admin.claim_school"), display_school(eligibility.current_employment.school)]
+        a << [t("student_loans.questions.admin.claim_school"), display_school(eligibility.selected_employment.school)]
```

Comment:
Are you planning to squash this commit into the [one where `current_employment` was introduced](https://github.com/DFE-Digital/dfe-teachers-payment-service/pull/454/commits/8d60a39b3d31029d394e9211e2eb794fdddc5ade)?

Reply:
Let's do this @robbiepaul.